_posts/2018-08-30-disability-at-rustfest.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: post
+title: "Disability @ RustFest"
+#socialImageSrc: "/assets/posts/rome.jpg"
+authors:
+  - skade
+  - katsel
+---
+
+RustFest Rome has a special program for disabled Rustaceans. First of all, check our [accessibility statement](https://rome.rustfest.eu/accessibility/) for general information about the accessibility of our venue and our already present accessibility features, such as live transcription. If there is anything missing that you need, please get in touch with us as early as possible. Also, you may want to read about [Diversity @ RustFest](https://rome.rustfest.eu/diversity).
+
+Also, we have marked 5 of our free tickets explicitly for disabled people. Please go to [TODO: diversitytickets.org-link] to apply. "Disability" includes physical as well as mental disabilities. These tickets include entrance to the conference plus travel for you and a second person, should you require assistance. Please indicate on the application if this is the case. It won't influence your chances of acceptance, just quicken the process.
```

Comment:
The `Also,` is unnecessary.

Reply:
> Also, we have marked 5 of our free tickets explicitly for disabled people.

"We have marked 5 of our free diversity tickets explicitly for disabled people."

> "Disability" includes physical as well as mental disabilities.

"Disability includes physical as well as non-physical disabilities and neurodivergence."